Pod/Classes/CDTQQueryValidator.m

```diff
@@ -11,9 +11,23 @@
 
 @implementation CDTQQueryValidator
 
-static const NSString *AND = @"$and";
-static const NSString *OR = @"$or";
-static const NSString *EQ = @"$eq";
+static NSString *const AND = @"$and";
+static NSString *const OR = @"$or";
+static NSString *const EQ = @"$eq";
+static NSString *const NOT = @"$not";
+static NSString *const NE = @"$ne";
+
+// notOperators dictionary is used for operator shorthand processing.
+// Presently only $ne is supported.  More to come soon...
++ (NSDictionary *)getNotOperators
+{
+    static NSDictionary *notOperators = nil;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        notOperators = @{ NE : EQ };
+    });
+    return notOperators;
+}
```

Comment:
This was what I set out to accomplish initially but apparently initializing a dictionary as a constant is not syntactically valid in Objective C.  It raises the following error: `Initializer element is not a compile-time constant`.  It seems that the only alternative to this is to set the dictionary as a variable in the functions that use it but then the dictionary would need to be maintained in multiple places.  Since the list will grow (somewhat) as we begin to add more features I chose to use the dispatch once approach wrapped in a function because it allows us to manage a single "class-wide" representation of the $not operators dictionary.